lib/Sema/CSDiag.cpp

```diff
@@ -5333,9 +5333,10 @@ bool FailureDiagnosis::visitApplyExpr(ApplyExpr *callExpr) {
         CS.DynamicCallableCache[fnType->getCanonicalType()].isValid();
 
     auto *nominal = fnType->getAnyNominal();
-    auto hasCallDecls = nominal &&
+    auto hasCallableMethods = nominal &&
```

Comment:
I don't believe it should be called a "callable method". Every method is callable, so this is confusing. "Call method" would be more clear.

Reply:
Renamed occurrences of "callable method" to "call method".